tests/rest/client/test_rooms.py

```diff
@@ -966,6 +970,81 @@ async def user_may_join_room(
         self.helper.join(self.room3, self.user2, expect_code=403, tok=self.tok2)
 
 
+class RoomAppserviceTsParamTestCase(RoomBase):
+    servlets = [
+        room.register_servlets,
+        synapse.rest.admin.register_servlets,
+        register.register_servlets,
+    ]
+
+    def prepare(self, reactor, clock, homeserver):
+        self.ts = 1
```

Comment:
it's a bit odd to make this a property of the test object, rather than a local variable.